functions/src/definitions/eventHandlers/onMessageUpdate.ts

```diff
@@ -2,21 +2,24 @@ import * as functions from "firebase-functions"
 import { respondToInstance } from "../common/responseUtils"
 import { Timestamp } from "firebase-admin/firestore"
 import { rationaliseMessage, anonymiseMessage } from "../common/genAI"
+import { onDocumentUpdated } from "firebase-functions/v2/firestore"
 
-const onMessageUpdate = functions
-  .region("asia-southeast1")
-  .runWith({
+const onMessageUpdateV2 = onDocumentUpdated(
+  {
+    document: "messages/{messageId}",
```

Comment:
```suggestion
    document: "/messages/{messageId}",
```

Reply:
The examples in the documentation seem to go without the leading slash so I went with it!